tests/plugins/icos/test_reader.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+
+from pyaerocom import const
+from pyaerocom.plugins.mep.reader import ReadMEP
+from pyaerocom.plugins.icos.reader import ReadICOS
+from tests.conftest import lustre_unavail
+
+try:
+    ICOS_PATH = Path(const.OBSLOCS_UNGRIDDED[const.ICOS_NAME])
+except KeyError:
+    pytest.mark.skip(reason=f"ICOS path not initialised due to non existence in CI")
```

Comment:
if all tests need to be decorated with `@lustre_unavail`
all test in the module can be skipped as follows:
``` python
if not const.has_access_lustre:
    pytestmark = pytest.skip("Skipping tests that require access to AEROCOM database on METNo servers")

try:
    ICOS_PATH = Path(const.OBSLOCS_UNGRIDDED[const.ICOS_NAME])
except KeyError:
    pytestmark = pytest.skip("ICOS path not initialised due to non existence in CI")
```

but I suspect that `pytestmark` from the exception is enough

Reply:
#910